src/tlo/core.py

```diff
@@ -138,6 +141,8 @@ class Property(Specifiable):
         float: float("nan"),
         "category": float("nan"),
         object: float("nan"),
+        np.uint32: 0,
+        np.uint64: 0,
```

Comment:
```suggestion
```

Unless we want `np.uint64` in case we switch to it in future?

Reply:
Yeah I was in two minds about whether to leave this in or not. I ended up doing as I think having a default value set for `np.uint64` columns means any columns of that type would have a (possibly) sensible default irrespective of if they are bitset related or not, and it would mean not needing to remember to separately update the default map as well if we switched back to 64-bit representation in future. However it is a bit messy and by the same argument we should potentially also add defaults for `np.uint16` and `np.unit8` which we currently don't so probably on balance best to just remove.